cupyx/scipy/signal/_signaltools.py

```diff
@@ -411,6 +411,40 @@ def _correlate2d(in1, in2, mode, boundary, fillvalue, convolution=False):
     return _st_core._direct_correlate(in1, in2, mode, in1.dtype, convolution,
                                       boundary, fillvalue, not convolution)
 
+def correlation_lags(in1_len, in2_len, mode='full'):
+    # calculate lag ranges in different modes of operation
```

Comment:
Please add a docstring for docs generation

Reply:
Docstring has been added.